3dtools/README.md

```diff
@@ -47,6 +47,10 @@ Some basic parameters should be told before start the program.
 >
 User can also just run the *main.py* in python consoler. In this case, a default server "dell-poweredge-r720xd.rear.png" will be shown.
 
```

Comment:
If you can review the whole readme please, I think some parts are not clear and/or the English is not correct.